client/src/CodeChatEditor.mts

```diff
@@ -33,17 +33,30 @@ import "./graphviz-webcomponent-setup.mts";
 import "./graphviz-webcomponent/index.min.mjs";
 import { html_beautify } from "js-beautify";
 import { tinymce, tinymce_init } from "./tinymce-webpack.mjs";
+import { tables, taskListItems } from 'turndown-plugin-gfm';
+import TurndownService from "turndown";
 
-// Not exactly an import, but this seems like the place to instantiate this
-const TurndownService = require("turndown");
-const turndownService = new TurndownService();
 
 // ### CSS
 import "./../static/css/CodeChatEditor.css";
 
 // Initialization
 // --------------
 //
+// Instantiate [turndown](https://github.com/mixmark-io/turndown) for HTML to Markdown conversion
+const turndownService = new TurndownService();
+
+// Add the plugins from [turndown-plugin-gfm](https://github.com/mixmark-io/turndown-plugin-gfm) to enable conversions for tables & ordered and unordered lists
+turndownService.use([tables, taskListItems]);
+
+// To add the conversion from HTML to Markdown for strikethroughs, a new rule can be added using turndown's addRule() function. For HTML strikethrough tags, <s>, the conversion will add '~~' surrounding the content within the open and closing HTML tags.
```

Comment:
There was an issue with this plugin for strikethroughs. It only added a single tilde around the strikethrough text rather than two tildes. The forked/updated version of the plugin that we're now using fixes this issue so this code has now been removed.